### DISCUSSION TITLE
Documentation for a specific version

### DISCUSSION THREAD

**Discussion by JeremyBradshaw7** [invertase/react-native-firebase#7605] [Q&A] [answered]
I can't upgrade to the latest v18 because I'd need to update my MacOS, so I'm trying v16. Where can I find the installaion documentation for v16? The installation steps can change each major version but there's no version selector at https://rnfirebase.io/ as far as I can see.

**Comment by mikehardy** [comment, marked as answer]
@JeremyBradshaw7 that version is not supported. I can't recommend you use it, in fact I can only recommend that you use current stable version as old versions will likely not work correctly on new versions of the mobile operating systems

Because of that we don't maintain old versions of documentation, but you could use github to [surf the docs tree for that tag ](https://github.com/invertase/react-native-firebase/tree/v16.7.0/docs)

If you're open to an alternative...

Assuming you can't update because the hardware is too old - I highly recommend you try https://github.com/dortania/OpenCore-Legacy-Patcher - I use it on a couple machines with no issues, and I do so precisely because I need them as test rigs for my maintainer work here on react-native-firebase

**Reply by JeremyBradshaw7** [reply]
Thanks @mikehardy - I've taken the first steps in using the open core legacy patcher as a result of your recommendation on another ticket, but not quite completed it yet.

**Reply by mikehardy** [reply]
cool - sincere best wishes with it. I've been using it a couple years now (I have one *really* old machine), and just extended it to another machine that finally stopped receiving updates recently.

Only thing I can think of with regard to doing it safely is that you may want to make sure that the actual OCLP app is present on the machine (not just on, for instance, a USB key, or just installed then deleted). Out of the 15 or so OS updates I've done since starting it there was one that needed an OCLP reinstall (not just a re-patch) after rebooting, and it was a pain because the network was one of the things depending on it. USB key to move the file over saved me, but even easier if I just leave the OCLP download on the desktop ready if needed

Not intending for RNFB discussions to become an OCLP support forum, but it has been such a lifesaver and it's keeping some machines out of the landfill, so I'm a fan. Cheers
